### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by HardipinderS**
# Hey, I just made a Pull Request!

## Description
Makefile not working while running make livehtml and update on the docs for windows user

## Motivation and Context
removing -p, as it is not working. Rather having it run directly as addition of --open-browser and --port fixes that

## Have you tested this? If so, how?
ran make check

## Checklist for PR author(s)
**check it anyway**

- [ ] Changes are covered by unit tests (no major decrease in code coverage %).
- [ ] All tests pass.
- [ ] Docstring coverage is **100%** via `interrogate drf_user` (I mean, we _should_ set a good example :smile:).
- [ ] Updates to documentation:
    - [ ] Document any relevant additions/changes in `README.md`.
    - [ ] Manually update **both** the `README.md` _and_ `docs/index.rst` for any new changes.
    - [ ] Any changed/added classes/methods/functions have appropriate `versionadded`, `versionchanged`, or `deprecated` [directives](http://www.sphinx-doc.org/en/stable/markup/para.html#directive-versionadded).  Find the appropriate next version in the project's [``__init__.py``](https://github.com/101loop/drf-user/blob/master/drf_user/__init__.py) file.

## Release note

```release-note

```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -166,13 +166,13 @@ like [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv),
...
-$ git clone git@github.com:101loop/drf-user.git
+$ git clone git@github.com:<your-username>/drf-user.git
 ```
 
 or if you want to use git via `https`:
 
 ```sh
-$ git clone https://github.com/101loop/drf-user.git
+$ git clone https://github.com/<your-github-username>/browser-history.git
 ```
```
Comment (sumit4613) [inline CONTRIBUTING.md:176]: @HardipinderS Actually, we have to add another section for using forked copy of this repo.
You can take reference from this link https://github.com/101Loop/APIManager-Flutter/blob/master/CONTRIBUTING.md

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -185,6 +185,15 @@ use [pre-commit](https://pre-commit.com/) hooks:
...
 
+For Windows User:
+```
+# install chocolatey and run "choco install make" to use make functions
+(env) $ cd drf-user
+(env) $ pip install -r requirements-dev.txt
+(env) $ pip install -e .
+pre-commit install
+```
+
```
Comment (sumit4613) [inline CONTRIBUTING.md:196]: @HardipinderS If you can put these steps in a different section about `setting up in windows system` that would be much better. Please try to put these details in separate section.

File: docs/Makefile [hunk 3]
```diff
@@ -24,7 +24,7 @@ html:
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html"
 
 livehtml:
-	sphinx-autobuild -p $(PORT) -b html $(SOURCEDIR) $(BUILDDIR)/html $(O)
+	sphinx-autobuild --port $(PORT) -b html $(SOURCEDIR) $(BUILDDIR)/html $(O) --open-browser
```
Comment (sumit4613) [inline docs/Makefile:27]: Didn't know about this, thanks for updating this.

**Review by sumit4613 — CHANGES_REQUESTED** [review]
@HardipinderS Thank you for taking out time and working on this project. 
Please update these details a little bit. And if you have any doubts or queries please put it here will help you in solving them.

Thanks!

File: CONTRIBUTING.md (hunk 1 above)
Comment (HardipinderS) [reply CONTRIBUTING.md:180]: Added changes I've pushed hopefully it will get added

File: CONTRIBUTING.md (hunk 2 above)
Comment (HardipinderS) [reply CONTRIBUTING.md]: Added

File: CONTRIBUTING.md (hunk 1 above)
Comment (HardipinderS) [reply CONTRIBUTING.md:180]: Added the link directly wasn't sure how much I could add

File: CONTRIBUTING.md (hunk 2 above)
Comment (HardipinderS) [reply CONTRIBUTING.md]: Added the new changes separately for Linux and Windows

File: docs/Makefile (hunk 3 above)
Comment (HardipinderS) [reply docs/Makefile:27]: happy it helped :)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -163,16 +163,20 @@ like [pyenv-virtualenv](https://github.com/pyenv/pyenv-virtualenv),
 [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/), and
 [pyenv-virtualenvwrapper](https://github.com/pyenv/pyenv-virtualenvwrapper).
 
+for git related issues refer to https://github.com/101Loop/APIManager-Flutter/blob/master/CONTRIBUTING.md
```
Comment (sumit4613) [inline CONTRIBUTING.md:166]: @HardipinderS I told you to take reference from that link and update this contributing guide on how to work on a forked version of this copy. So, please update the documentation guide how a user can contribute using a forked copy of this repo. We don't want users redirect to some other pages.

**Review by sumit4613 — CHANGES_REQUESTED** [review]
@HardipinderS Please work on forking section of the contributing guide first then we'll work on other sections.

Thanks for working on this!

File: CONTRIBUTING.md (hunk 4 above)
Comment (HardipinderS) [reply CONTRIBUTING.md:166]: I'm not sure what and how much exactly is to be referred and added in that part, so gave the link. i would request you to please look into that part, as I'm not that sure what exactly is required

**Comment by HardipinderS** [comment]
won't be helping out any more

**Pull request closed without merging**